src/main/scala/ch/openolitor/core/domain/EntityStoreView.scala

```diff
@@ -66,6 +68,28 @@ trait EntityStoreView extends PersistentView with DBEvolutionReference with Lazy
 
   import EntityStore._
 
+  final case class Terminate()
+  var failures = 0;
+
+  override protected def onReplayError(cause: Throwable): Unit = {
+    super.onReplayError(cause)
+    failures = failures + 1;
+    if (failures == 1) {
+      val system = ActorSystem("oo-system")
+      system.scheduler.scheduleOnce(Duration.create(1, TimeUnit.HOURS), context.self, Terminate())
+    }
+  }
+
+  def terminateSystem(): Unit = {
+    if (failures > 200) {
+      log.error("The system was not able to recover. A forced termination is called")
+      context.system.terminate()
```

Comment:
This will shut down the `ActorSystem` of the mandant and then leave it like that.
I think we should send a message to errbit that we're going to shutdown and force a restart. Then send a message to self in one minute and do a System.exit('R') ('R' for recovery failed or so) to force Cloud Foundry to restart our container.

Reply:
all done